common/components/controllers/EditProfileController.jsx

```diff
@@ -160,7 +160,7 @@ class EditProfileController extends React.PureComponent<{||},State> {
       <div className="wrapper-gray">
         <div className="container">
           <form action={`/api/user/edit/${window.DLAB_GLOBAL_CONTEXT.userID}/`} method="post">
-            <div className="EditProjectForm-root">
+            <div className="EditProjectForm-root create-form white-bg container-fluid">
```

Comment:
I don't think we need the `container-fluid` class here. Line 161 already has this entire section in a `.container` so we're good already.